### PR TITLE
MTDSA-26823: Rename and add subtasks using ACs - Individuals Capital Gains Income API

### PR DESCRIPTION
**Pull request by DavouJ**

### CODE REVIEW

**Review by MarkAKelly — COMMENTED** [review]
Would you mind adding a description of the change in the title please @DavouJ

File: resources/public/api/conf/common/errors.md [hunk 1]
```diff
@@ -1,58 +1,2 @@
...
-{
-"code": "FORMAT_STRING_NAME",
-"message": "The provided field is not valid",
-"paths": [ "/arrayName/0/fieldName3", "/arrayName/1/fieldName3" ]
-}
-]
-}`
\ No newline at end of file
+Errors returned by this API are listed in the Response section of each endpoint. 
+Read the [reference guide](https://developer.service.hmrc.gov.uk/api-documentation/docs/reference-guide#errors) for more information about errors.
```
Comment (AhmadZNaeem) [inline resources/public/api/conf/common/errors.md:2]: Environment specific url can be used: `[reference guide](/api-documentation/docs/reference-guide#errors)`

File: resources/public/api/conf/common/testing.md [hunk 2]
```diff
@@ -1,20 +1,6 @@
-You can use the sandbox environment to <a href="/api-documentation/docs/testing">test this API</a>. You can use the <a href="/api-documentation/docs/api/service/api-platform-test-user/1.0">Create Test User API</a> or it's frontend service to create test users.
+You can use the Sandbox environment to [test this API](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing). You can test different scenarios in the Sandbox by passing the
```
Comment (AhmadZNaeem) [inline resources/public/api/conf/common/testing.md:1]: Environment specific url can be used `[test this API](/api-documentation/docs/testing)`

File: resources/public/api/conf/common/testing.md [hunk 3]
```diff
@@ -1,20 +1,6 @@
...
-particular types of submissions. For each type you can POST (or PUT) to submit or amend data, GET to retrieve or list
-data and DELETE to delete data. For example, with a STATEFUL gov test scenario a retrieval will return data based on
-what you submitted.
-
-The following groups are stateful in the sandbox:
-
-- Capital Gains on Residential Property Disposals
-- Other Capital Gains and Disposals
\ No newline at end of file
+If you have a specific testing need that is not supported in the Sandbox, [contact our Software Developers Support Team](https://developer.service.hmrc.gov.uk/developer/support).
```
Comment (AhmadZNaeem) [inline resources/public/api/conf/common/testing.md:5]: Environment specific url can be used `[contact our Software Developers Support Team](/developer/support)`

File: resources/public/api/conf/common/versioning.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+When [backwards-incompatible](https://developer.service.hmrc.gov.uk/guides/income-tax-mtd-end-to-end-service-guide/documentation/how-to-integrate.html#breaking-changes) changes are made to this API, we release a new version. 
+Read the [reference guide](https://developer.service.hmrc.gov.uk/api-documentation/docs/reference-guide#versioning) for more information about API versioning.
```
Comment (AhmadZNaeem) [inline resources/public/api/conf/common/versioning.md:2]: Environment specific url can be used: `[reference guide](/api-documentation/docs/reference-guide#versioning)`

**Comment by AhmadZNaeem** [comment]
I'm away on leave so can someone else approve this ticket once comments have been addressed. Thank you

File: resources/public/api/conf/common/testing.md [hunk 5]
```diff
@@ -1,20 +1,6 @@
...
-particular types of submissions. For each type you can POST (or PUT) to submit or amend data, GET to retrieve or list
-data and DELETE to delete data. For example, with a STATEFUL gov test scenario a retrieval will return data based on
-what you submitted.
-
-The following groups are stateful in the sandbox:
-
-- Capital Gains on Residential Property Disposals
-- Other Capital Gains and Disposals
\ No newline at end of file
+If you have a specific testing need that is not supported in the Sandbox, [contact our Software Developers Support Team](]/developer/support).
```
Comment (benryan1790) [inline resources/public/api/conf/common/testing.md:5]: errant closing square bracket in the link

**Review by benryan1790 — CHANGES_REQUESTED** [review]

**Review by benryan1790 — APPROVED** [review]

**Pull request merged**